policy/release/java.rego

```diff
@@ -25,7 +31,14 @@ import future.keywords.in
 #       - redhat
 #       - rebuilt
 deny[result] {
-	results := lib.results_named(lib.java_sbom_component_count_result_name)
+	java_results := lib.results_named(lib.java_sbom_component_count_result_name)
+
+	results := [result |
+		result := java_results[_]
+		result[lib.key_bundle]
+		bundle := result[lib.key_bundle]
+		bundles.is_acceptable(bundle)
```

Comment:
This would still allow a trusted bundle to include a task that spoofs the results, right? I think that's ok, just clarifying.

Reply:
Yes, otherwise we'd have to link a Task bundle to a task to a result, which would be difficult with the current Task bundles (having multiple tasks within them at semi-random). Wrote [a comment](https://issues.redhat.com/browse/HACBS-995?focusedCommentId=21042997&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-21042997) on this on the issue.